app/controllers/reviews_controller.rb

```diff
@@ -1,63 +1,110 @@
 class ReviewsController < ApplicationController
   before_action :set_review, only: [:show, :edit, :update, :destroy]
 
-  # GET /reviews
-  # GET /reviews.json
+  # GET /restaurants/:restaurant_id/reviews
+  # GET /restaurants/:restaurant_id/reviews.xml
   def index
-    @reviews = Review.all
+    #1st you retrieve the restaurant thanks to params[:restaurant_id]
+    restaurant = Restaurant.find(params[:restaurant_id])
```

Comment:
If you want, you can add `before_action :find_restaurant` at the top of the controller and then add this method somewhere:

```
def find_restaurant
  @restaurant = Restaurant.find params[: restaurant_id]
end
```

And then `@restaurant` will be available in all the methods.

Reply:
Cool tip thanks :)